logging/zap/grpclogger.go

```diff
@@ -44,3 +45,80 @@ func (l *zapGrpcLogger) Printf(format string, args ...interface{}) {
 func (l *zapGrpcLogger) Println(args ...interface{}) {
 	l.logger.Info(fmt.Sprint(args...))
 }
+
+// GrpcLoggerV2Options allows you to set grpclogger options when using ReplaceGrpcLoggerV2.
+type GrpcLoggerV2Options struct {
+	verbosity int
+}
+
+// DefaultGrpcLoggerV2Options is a set of default options for ReplaceGrpcLoggerV2.
+var DefaultGrpcLoggerV2Options = &GrpcLoggerV2Options{}
+
+// ReplaceGrpcLoggerV2 replaces the grpc_log.LoggerV2 with the input zap.Logger.
+// This logger allows you to set grpc verbosity level.
+func ReplaceGrpcLoggerV2(logger *zap.Logger, opts *GrpcLoggerV2Options) {
```

Comment:
Lets change this to `ReplaceLoggerV2(logger *zap.Logger)` this will match the upstream here https://github.com/grpc/grpc-go/blob/master/grpclog/loggerv2.go#L101

Reply:
Add a second `ReplaceLoggerV2WithVerbosity` with the option to pass in `v int` as https://github.com/grpc/grpc-go/blob/master/grpclog/loggerv2.go#L107 again matching upstream

Reply:
Change `ReplaceLoggerV2` to call `ReplaceLoggerV2WithVerbosity(logger , 0)` see https://github.com/grpc/grpc-go/blob/master/grpclog/loggerv2.go#L102

Reply:
🤔 I can do that! Want to drop the Grpc from the name?